src/main.rs

```diff
@@ -1,6 +1,50 @@
 // SPDX-License-Identifier: Apache-2.0
 
-//! Loads keeps from their backends.
+//! This crate provides the `enarx-keepldr` executable which loads `static-pie`
+//! binaries into an Enarx Keep - that is a hardware isolated environment using
+//! technologies such as Intel SGX or AMD SEV.
+//!
+//! # Install Dependencies
+//!
+//! ## Fedora
+//!
+//!     $ sudo dnf install git curl gcc pkg-config openssl-devel musl-gcc
```

Comment:
````suggestion
//! ```console
//! $ sudo dnf install git curl gcc pkg-config openssl-devel musl-gcc
//! ```
````

Reply:
and the rest likewise ... just a suggestion